net-vpn/tor/files/tor.initd-r10

```diff
@@ -1,22 +1,27 @@
 #!/sbin/openrc-run
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 command=/usr/bin/tor
-pidfile=/run/tor/tor.pid
-command_args="--hush --runasdaemon 1 --pidfile \"${pidfile}\""
+pidfile=/run/tor/${RC_SVCNAME}.pid
+command_args="--hush"
+command_args_background="--runasdaemon 1 --pidfile \"${pidfile}\""
 retry=${GRACEFUL_TIMEOUT:-60}
 stopsig=INT
 command_progress=yes
 
+depend() {
+	after net
```

Comment:
_If_ we're going to do this, it needs a comment, and the commit message needs justification. You should also `Bug:` tag https://bugs.gentoo.org/619910 in the commit message.

Reply:
What justification?